rclcpp_lifecycle/src/lifecycle_node.cpp

```diff
@@ -306,46 +306,93 @@ LifecycleNode::trigger_transition(const Transition & transition)
 }
 
 const State &
+LifecycleNode::trigger_transition(const Transition & transition, rcl_ret_t & error)
+{
+  return trigger_transition(transition.id(), error);
+}
+
+const State &
 LifecycleNode::trigger_transition(uint8_t transition_id)
 {
   return impl_->trigger_transition(transition_id);
 }
 
 const State &
+LifecycleNode::trigger_transition(uint8_t transition_id, rcl_ret_t & error)
```

Comment:
This signature doesn't match the header (`rcl_ret_t` vs. `rcl_lifecycle_ret_t`)?